apps/prairielearn/src/question-servers/freeform.js

```diff
@@ -702,9 +720,9 @@ async function legacyTraverseQuestionAndExecuteFunctions(phase, codeCaller, data
 
         const elementFile = getElementController(elementName, context);
         // Populate the extensions used by this element
-        data.extensions = [];
+        /** @type {any} */ (data).extensions = [];
         if (Object.prototype.hasOwnProperty.call(context.course_element_extensions, elementName)) {
-          data.extensions = context.course_element_extensions[elementName];
+          /** @type {any} */ (data).extensions = context.course_element_extensions[elementName];
```

Comment:
I noticed this on the other PR -- I think the two types are empty list, and a dictionary. This could benefit from stronger typing so that it's picked up.